composer.json

```diff
@@ -56,14 +56,12 @@
         "jcalderonzumba/gastonjs": "^1.2",
         "jcalderonzumba/mink-phantomjs-driver": "^0.3.3",
         "metadrop/behat-contexts": "dev-fb-nuvoleweb-dev-i45-behat-4.x",
-        "metadrop/grumphp-drupal-check": "~0.1",
+        "metadrop/grumphp-drupal-check": "^0.3.0",
         "metadrop/scripthor": "1.x-dev",
-        "mglaman/drupal-check": "^1.1",
         "mikey179/vfsstream": "^1.6",
         "pdepend/pdepend": "^2.8",
         "phpmd/phpmd": "^2.9",
-        "sebastian/phpcpd": "^5.0",
-        "vijaycs85/drupal-quality-checker": "^1.3.0"
+        "phpro/grumphp": "^1.3.0"
```

Comment:
I would remove this dependency because is already declared on phpro/grumphp
https://github.com/Metadrop/grumphp-drupal-check/blob/master/composer.json

Reply:
I think I didn't submit my last comment where I explained why I don't want remove it.

I understand that if grumphp is indirectly required so we could remove it, reducing maintenance. 

However, we need grumphp in our project with or without the grumphp drupal check  plugin. If we need to use grumphp for more reasons than using grumphp-drupal-check (phpcs, phpcpd, git commit msg), then why should we define grumphp as an indirect dependency. I think it should be direct dependency.

Reply:
I've been clarified by David in a call, in the last commit I explain the reason explaining why we drop the dependency.